hbase-balancer/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -2176,13 +1606,9 @@ private static void debugLogMutation(Mutation p) throws IOException {
   }
 
   private static Put addSequenceNum(Put p, long openSeqNum, int replicaId) throws IOException {
-    return p.add(CellBuilderFactory.create(CellBuilderType.SHALLOW_COPY)
-              .setRow(p.getRow())
-              .setFamily(HConstants.CATALOG_FAMILY)
-              .setQualifier(getSeqNumColumn(replicaId))
-              .setTimestamp(p.getTimestamp())
-              .setType(Type.Put)
-              .setValue(Bytes.toBytes(openSeqNum))
-              .build());
+    return p.add(CellBuilderFactory.create(CellBuilderType.SHALLOW_COPY).setRow(p.getRow())
+      .setFamily(HConstants.CATALOG_FAMILY)
+      .setQualifier(CatalogFamilyFormat.getSeqNumColumn(replicaId)).setTimestamp(p.getTimestamp())
+      .setType(Type.Put).setValue(Bytes.toBytes(openSeqNum)).build());
   }
 }
```

Comment:
Trying to figure the prinicipal to apply figuring when to put something in MTA and when to put it in CMTA? Thanks.